.github/workflows/build-test.yml

```diff
@@ -118,11 +117,54 @@ jobs:
           token: ${{ secrets.ACCESS_TOKEN }}
           type: ${{ job.status }}
           url: ${{ secrets.SLACK_WEBHOOK_URL }}
-
+  docker-build:
+    runs-on: ubuntu-latest
+    needs: [unit-tests]
+    steps:
+      - name: Clone airnode
+        uses: actions/checkout@v2
+        with:
+          # Required for changesets check. See: https://github.com/changesets/changesets/issues/517#issuecomment-813282523
+          fetch-depth: 0
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ env.TARGET_NODE_VERSION }}
+      - name: restore lerna
+        id: cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            node_modules
+            */*/node_modules
+          key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
+      - name: Install Dependencies
+        run: yarn run bootstrap
+      - name: yarn docker
+        env:
+          DOCKER_BUILDKIT: 1
+        run: yarn docker
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Retag docker images to dev repositories
+        run: |
+          docker tag api3/airnode-artifacts:latest api3/airnode-artifacts-dev:${{ github.sha }}
+          docker tag api3/airnode-deployer:latest api3/airnode-deployer-dev:${{ github.sha }}
+          docker tag api3/airnode-client:latest api3/airnode-client-dev:${{ github.sha }}
+          docker tag api3/airnode-admin:latest api3/airnode-admin-dev:${{ github.sha }}
```

Comment:
Is there a limit to how many builds can be hosted on Docker Hub? This will generate a lot over time.

Reply:
There is no limit :laughing: